code-scanning/clj-watson.yml

```diff
@@ -0,0 +1,48 @@
+# This workflow uses actions that are not certified by GitHub.
```

Comment:
Please add some comments at top which describes what this workflow does and if it requires some pre-requisites

Reply:
added comments